apiserver/plane/app/views/issue/base.py

```
@@ -1285,7 +1278,7 @@ def get(self, request, slug):
 
 class SearchSingleValueAPI(BaseAPIView):
     model = Issue
-    allowed_fields = ["hub_code", "trip_reference_number", "reference_number", "worker_code", "vendor_code","customer_code"]
+    allowed_fields = ["hub_code", "trip_reference_number", "reference_number", "worker_code", "vendor_code", "customer_code", "business_type"]
```

Pylint (3.3.7)

[convention] Line too long (142/100)

(C0301)